docs/docs/graphql-reference.md

```diff
@@ -14,74 +14,136 @@ For more information, read about [why Gatsby uses GraphQL](/docs/why-gatsby-uses
 
 Start with the basics, pulling up the site `title` from your `gatsby-config.js`'s `siteMetadata`. Here the query is on the left and the results are on the right.
 
-<iframe
+<GraphiQLEmbed
   title="A basic query"
-  src="https://711808k40x.sse.codesandbox.io/___graphql?query=%7B%0A%20%20site%20%7B%0A%20%20%20%20siteMetadata%20%7B%0A%20%20%20%20%20%20title%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A&explorerIsOpen=false"
-  width="600"
-  height="400"
-  loading="lazy"
-></iframe>
+  url="https://711808k40x.sse.codesandbox.io/___graphql"
+  query={`{
+  site {
+    siteMetadata {
+      title
+    }
+  }
+}
+`}
+/>
 
 Try editing the query to include the `description` from `siteMetadata`. When typing in the query editor you can use `Ctrl + Space` to see autocomplete options and `Ctrl + Enter` to run the current query.
 
 ## A longer query
 
 Gatsby structures its content as collections of `nodes`, which are connected to each other with `edges`. In this query you ask for the total count of plugins in this Gatsby site, along with specific information about each one.
 
-<iframe
+<GraphiQLEmbed
   title="A longer query"
-  src="https://711808k40x.sse.codesandbox.io/___graphql?query=%7B%0A%20%20allSitePlugin%20%7B%0A%20%20%20%20totalCount%0A%20%20%20%20edges%20%7B%0A%20%20%20%20%20%20node%20%7B%0A%20%20%20%20%20%20%20%20name%0A%20%20%20%20%20%20%20%20version%0A%20%20%20%20%20%20%20%20packageJson%20%7B%0A%20%20%20%20%20%20%20%20%20%20description%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A&explorerIsOpen=false"
-  width="600"
-  height="400"
-  loading="lazy"
-></iframe>
+  url="https://711808k40x.sse.codesandbox.io/___graphql"
+  query={`{
+  allSitePlugin {
+    totalCount
+    edges {
+      node {
+        name
+        version
+        packageJson {
+          description
+        }
+      }
+    }
+  }
+}
+`}
+/>
 
 Try using the editor's autocomplete (`Ctrl + Space`) to get extended details from the `packageJson` nodes.
 
 If you're using Gatsby version `2.2.0` or later, you can remove `edges` and `node` from your query and replace it with `nodes`. The query will still work and the returned object will reflect the `nodes` structure.
 
-<iframe
+<GraphiQLEmbed
   title="A longer query with nodes"
-  src="https://711808k40x.sse.codesandbox.io/___graphql?query=%7B%0A%20%20allSitePlugin%20%7B%0A%20%20%20%20totalCount%0A%20%20%20%20nodes%20%7B%0A%20%20%20%20%20%20%20%20name%0A%20%20%20%20%20%20%20%20version%0A%20%20%20%20%20%20%20%20packageJson%20%7B%0A%20%20%20%20%20%20%20%20%20%20description%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D&explorerIsOpen=false"
-  width="600"
-  height="400"
-></iframe>
+  url="https://711808k40x.sse.codesandbox.io/___graphql"
```

Comment:
could we maybe make this the default value of the `url` prop so that folks don't have to specify it every time?